### PR TITLE
DDF-906 - added any/specific toggle to metadata content type facet.

### PR DESCRIPTION
**Pull request by McLucas**
-This toggle will help tell the user when "any" content type is being used
vs when specific content types are in your query.

### CODE REVIEW

**Comment by shaundmorris** [comment]
@stustison or @pklinef any final comments here?

**Comment by McLucas** [comment]
what colors do you guys want me to use.  
sidebyside

**Comment by stustison** [comment]
I would say to use the one on the left to be consistent with the rest of the UI.

**Comment by McLucas** [comment]
I believe @pklinef  is in the same boat.  He mentioned using the left to match the rest of the UI.  LEFT IT IS!  Updated!

**Comment by pklinef** [comment]
Yeah, I think the one on the left is more consistent.  We only want the main action button to be blue like that too.

File: search-ui/standard/src/main/webapp/js/view/filter/FacetItem.view.js [hunk 1]
```diff
@@ -28,7 +29,40 @@ define([
...
+                wreqr.vent.trigger('anyFacetClicked', this.model.get('fieldName'));
+            },
+            toggleState: function(){
+                this.stateModel.set({
+                    state: this.stateModel.get('state') === 'any' ? 'specific' : 'any'
+                });
+            },
+            stateModelChanged: function(){
+                var state = this.stateModel.get('state');
+                this.$('tbody').attr('role',state);
```
Comment (pklinef) [inline search-ui/standard/src/main/webapp/js/view/filter/FacetItem.view.js]: I get nervous when I see this many calls out to jQuery.  Was there not a good way to drive this through the templates?

Comment (McLucas) [inline search-ui/standard/src/main/webapp/js/view/filter/FacetItem.view.js]: it probably can be.  and it probably should.  I'll make some adjustments to move this logic to the template

Comment (McLucas) [inline search-ui/standard/src/main/webapp/js/view/filter/FacetItem.view.js]: fixed.

**Pull request merged**